bindings/python/benchmarks/benchmarks.py

```diff
@@ -107,6 +115,14 @@ class ProfileRead:
     of reading MongoDB data.
     """
 
+    # We need this because the naive methods don't always convert nested objects.
+    @staticmethod
+    def exercise_table(table):
+        [
+            [[n for n in i.values] if isinstance(i, pyarrow.ListScalar) else i for i in column]
+            for column in table.columns
+        ]
+
     def setup(self):
         db[collection_names[CUR_SIZE]].drop()
```

Comment:
This needs a guard for whether the collection exists.

Reply:
dropping a non-existant collection is fine in MongoDB:
```
>>> client.test.doesnotexist.drop()
>>> client.test.doesnotexist.drop()
```

Reply:
Oh I see what you mean:
```
                   db[collection_names[CUR_SIZE]].drop()
               KeyError: False
```

Reply:
Should be resolved now because we aren't using keys.